src/ProjectNeon/Assets/Features/Effects/ApplyOnShieldBelowValue.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Linq;
+using UnityEngine;
+
+public sealed class ApplyOnShieldBelowValue : ConditionalEffect
+{
+    private int _value;
+
+    public ApplyOnShieldBelowValue(Effect effect, int value) : base(effect)
+    {
+        _value = value;
+    }
+
+    public override bool Condition()
+    {
+        Debug.Log("Value: " + _value);
+        return Target.Members.Any(e => e.State[TemporalStatType.Shield] < _value );
```

Comment:
Is this being applied to only the Unshielded members? The target group is going to be All for this one.

Reply:
@SilasReinagel This effect is about appling only to unshielded members, why should we apply it to all targets?

Reply:
@paulodamaso The question wasn't answered.

Are you certain that this correctly applies to only Unshielded members, even with a Group Target (All/All as per the card), even if one shielded member is present?

The logic in this condition looks wrong.

Reply:
@SilasReinagel Actually not:
`PlayedCard.perform()` calls `Apply` for each `Action` it has, for each target. But it dismantle the `Target` array and calls apply for each individual target. That's why `Target.Members.Any`, in this case (inside an effect), will ever be an individual target.